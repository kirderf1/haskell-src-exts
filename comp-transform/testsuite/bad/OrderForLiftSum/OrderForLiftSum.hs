{-# LANGUAGE ComposableTypes #-}

piececategory A

eval -: Bool -> A -> Int

main = putStrLn $ "Det funkar!"
