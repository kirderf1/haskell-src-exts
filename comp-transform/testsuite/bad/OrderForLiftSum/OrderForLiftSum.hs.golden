Found category "A" in an unexpected spot in destructor "eval". Should only be allowed as first argument.
